handlers.go

```diff
@@ -557,7 +522,7 @@ func mkSelectString(params map[string]string) (string, []interface{}, error) {
 }
```

Comment:
i'm looking at that routine, getDeployments.  it is not clear to me how the technique used in this code is better than what my code does?  could you please be more explicit about what you are suggesting, and what its advantages are?